aspnetcore/fundamentals/error-handling.md

```diff
@@ -247,6 +247,8 @@ The automatic creation of a `ProblemDetails` can be customized using any of the
 2. Use a custom [`IProblemDetailsWriter`](#custom-iproblemdetailswriter)
 3. Call the [`IProblemDetailsService` in a middleware](#problem-details-from-middleware)
 
+***Note:*** When using a custom `IProblemDetailsWriter`, the custom `IProblemDetailsWriter` must be registered before calling <xref:Microsoft.Extensions.DependencyInjection.MvcServiceCollectionExtensions.AddRazorPages%2A>, <xref:Microsoft.Extensions.DependencyInjection.MvcServiceCollectionExtensions.AddControllers%2A>, or <xref:Microsoft.Extensions.DependencyInjection.MvcServiceCollectionExtensions.AddControllersWithViews%2A>.
```

Comment:
What do you think about adding it under https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/error-handling?branch=pr-en-us-29221&view=aspnetcore-8.0#custom-iproblemdetailswriter, call out that it must be registered before AddProblemDetails as well and maybe add an example of registration?